dataflowlauncher/launcher.py

```diff
@@ -40,9 +40,9 @@ def run(args, unknown_args, exec_path):
 
     """Generate parameters with config readers, update parameters with cli parsers """
     parameter_dict = get_jar_parameter_dict(config)
-    parameter_dict.update(get_updated_launch_params(config, args))
+    parameter_dict.update(get_updated_launch_params(parameter_dict, args))
```

Comment:
Any reason why we're passing in the `parameter_dict` instead of the `config`?

I don't think we were using the `config` previously. I guess it might be good to document the practice after we formulate it.